Cargo.toml

```diff
@@ -16,33 +16,32 @@ edition = "2018"
 members = ["proto"]
 
 [features]
-default = []
+default = ["repr-protobuf"]
 # Enable failpoints
 failpoint = ["fail"]
+repr-prost = ["raft-proto/repr-prost", "harness/repr-prost"]
+repr-protobuf = ["raft-proto/repr-protobuf", "harness/repr-protobuf"]
 
```

Comment:
Could we use `protobuf-codec` and `prost-codec`? It's a convention across a few crates and using the same feature names everywhere keeps things less confusing.

Reply:
Sure.